grouper/fe/templates/email/nonauditor_promoted_html.tmpl

```diff
@@ -0,0 +1,11 @@
+{% extends "email/base_html.tmpl" %}
+
+{% block subject %}Added as member to '{{ auditors_group_name }}'{% endblock %}
+
+{% block content %}
+<p>You are an approver (owner or manager) of one or more audited groups, but you were previously not an auditor.
+We have automatically made you an auditor by adding you to group '{{ auditors_group_name }}'.
+For more information, please see <a href="{{ url }}/help">Grouper Help</a>.
```

Comment:
Can we point them instead to the description on the auditor group page? The Grouper Help page doesn't seem like it'd be helpful here.
(Or maybe that means we should beef up the Grouper Help page...)

Reply:
eh later